.github/workflows/build-test.yml

```
@@ -360,6 +360,14 @@ jobs:
         run: |
           git config --global core.autocrlf false
 
+      - name: Run sccache-cache
+        uses: mozilla-actions/sccache-action@v0.0.3
+
+      - name: Set sccache env vars
+        run: |
+          echo "SCCACHE_GHA_ENABLED=true" >> "$GITHUB_ENV"
```

```suggestion
          echo "SCCACHE_GHA_ENABLED=on" >> "$GITHUB_ENV"
```
according to: https://github.com/mozilla/sccache/blob/main/docs/GHA.md